pages/docs/projects/tangle-network/overview.mdx

```diff
@@ -5,24 +5,63 @@ description: General overview of the Tangle Network.
 
 import { UseCasesArea, ParticipateArea, TestNetworkArea } from "../../../../components/UseCasesTangle";
 import FullWebbCTA from "../../../../components/FullWebbCTA";
+import SvgComponent from "../../../../components/TangleOverviewSVG.tsx";
```

Comment:
@thomivy I think you only added the Svg file to the page but you didn't place it anywhere. Will wait for this and we can merge.